openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -789,3 +793,42 @@ export const login = (username, password) => {
   cy.get('[id="password"]').should('be.visible').clear().type(password);
   cy.get('.ant-btn').contains('Login').should('be.visible').click();
 };
+
+export const addTeam = (TEAM_DETAILS) => {
+  interceptURL('GET', '/api/v1/teams*', 'addTeam');
+  //Fetching the add button and clicking on it
+  cy.get('button')
+    .find('span')
+    .contains('Add Team')
+    .should('be.visible')
+    .click();
+
+  verifyResponseStatusCode('@addTeam', 200);
+
+  //Entering team details
+  cy.get('[data-testid="name"]')
+    .should('exist')
+    .should('be.visible')
+    .type(TEAM_DETAILS.name);
+
+  cy.get('[data-testid="display-name"]')
+    .should('exist')
+    .should('be.visible')
+    .type(TEAM_DETAILS.displayName);
+
+  cy.get(descriptionBox)
+    .should('exist')
+    .should('be.visible')
+    .type(TEAM_DETAILS.description);
+  interceptURL('POST', '/api/v1/teams', 'saveTeam');
+  interceptURL('GET', '/api/v1/team*', 'createTeam');
+
```

Comment:
Can we add a test for the `Team type` field as well?

Reply:
Done 👍